src/Typesense/Key.ts

```diff
@@ -2,7 +2,7 @@ import ApiCall from "./ApiCall";
 import Keys from "./Keys";
 
 export interface KeyCreateSchema {
-  actions: string[];
+  actions: Actions[];
```

Comment:
We should add `string[]` as a fallback here, to prevent any backward incompatibility issues, now or in the future.